lib/bios/menus.robot

```diff
@@ -412,6 +413,48 @@ Reset To Defaults Tianocore
     Read From Terminal Until    ignore.
     Write Bare Into Terminal    y
 
+    # For platforms using PiKVM there is a need to turn on Serial Redirection otherwise most of tests might break.
+    IF    '${DUT_CONNECTION_METHOD}' == 'pikvm'
+        ${menu}=    Read From Terminal Until    Save screenshot
+        IF    '''Continue''' not in '''${menu}'''
+            ${menu}=    Go To Dasharo Main Screen
+        END
+        ${main_menu}=    Parse Menu Snapshot Into Construction    ${menu}    3    1
+        ${dasharo_menu}=    Enter Dasharo System Features    ${main_menu}
+        ${serial_menu}=    Enter Dasharo Submenu    ${dasharo_menu}    Serial Port Configuration
+        ${serial_state}=    Get Option State    ${serial_menu}    Enable COM0 Serial
+        IF    ${serial_state} != ${TRUE}
+            Set Option State    ${serial_menu}    Enable COM0 Serial    ${TRUE}
+        END
+        Press Key N Times    3    ${ESC}
+    END
+
+Go To Dasharo Main Screen
+    [Documentation]    This keyword will press ESC until it will detect the main Dasharo main screen.
+    FOR    ${i}    IN RANGE    0    10
```

Comment:
I tried and it needed 2 iterations to go back to the main menu with 1 nested screen, instead of one.

Reply:
Maciej created a seprate task for this issue: https://github.com/Dasharo/open-source-firmware-validation/issues/540